### ISSUE TITLE
Add BREAD for each role created

### ISSUE THREAD

**Issue by bfatoms** [thedevdojo/voyager#3430]
Everytime I create a role, and add permissions to them, I have to create a page for them by overriding the files, so lets say I created a payment page, I would like for the moderator to be able to edit certain fields on the payments page and also hide some fields on the browse page..
So basically my folder structure would look like

> slug/{role}/views-related ex. payments/moderator/edit-add.blade.php

, etc.. 
then the parent edit-add.blade.php which is in 

> slug/edit-add.blade.php

 will have a code
```
@if( auth()->user()->hasRole('moderator') )
  @include('views.vendor.payments.moderator.edit-add.blade.php')
@elseif ( ... ) 
 ......
@endif

```
Solution: Add a BREAD for each Role...
So Lets Say, I created a Page.. then create a role...
I would then go to database->edit bread..

On that page.. I think the design should look like a tabbed dividers.. like so
[ Superadmin ] [ Moderator ] [ User ]

So basically as long as I create a role, it should generate a tab in this location where you can edit the bread for the particular role...

I think this will be a great addition to Voyager.

**Comment by fletch3555** [comment]
Just override the view and add `@can('edit-[fieldName]`, $model)` (where `$model` is an instance of your model), then create entries in the permissions table for each of the fields you need to control.  Voyager's default policy will automatically convert the requested action to a permission following the `[action]_[BREAD slug]` pattern, so if you want to `edit-role` for `users`, you would have a permission called `edit-role_users`.  You can then assign that permission to the appropriate role(s), and not have to worry about creating a bunch of extra views

**Comment by emptynick** [comment]
Closing due to inactivity/possible solution provided.
